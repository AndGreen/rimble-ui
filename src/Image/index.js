import React from 'react';
import styled from 'styled-components'
import Box from '../Box'

const Image = styled(Box)`
  & {
    background: #F7F7F7;
  }
`

Image.defaultProps = {
  as: 'img',
  display: 'block',
  width: '100%'
}

Image.displayName = 'Image'

export default Image;
