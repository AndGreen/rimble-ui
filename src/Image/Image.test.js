import Image from './'

describe('Image component sanity', () => {
  it('has name', () => {
    expect(Image.displayName).toBe('Image')
  })

  it('matches default snapshot', () => {
    const component = render(<Image />)
    expect(component).toMatchSnapshot()
  })
})
