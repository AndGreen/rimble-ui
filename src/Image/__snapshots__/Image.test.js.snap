// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Image component sanity matches default snapshot 1`] = `
.c0 {
  background-color: #eee;
  width: 100%;
  display: block;
}

.c0 {
  box-sizing: border-box;
}

<img
  class="c0"
  display="block"
  width="100%"
>
  
</img>
`;
